src/components/planning/Planner.tsx

```diff
@@ -273,6 +293,15 @@ export default function Planner({
           ]}
         />
       </LabwareScanner>
+      {fetalSampleWarningLabware.length > 0 && (
+        <Warning
+          message={`The labware ${fetalSampleWarningLabware
+            .map((lw) => lw.barcode)
+            .join(",")} ${
+            fetalSampleWarningLabware.length > 1 ? " have" : " has"
+          } fetal waste samples collected less than ${FETAL_STORAGE_WEEKS} weeks. Please keep the fetal waste for ${FETAL_STORAGE_WEEKS} weeks.`}
```

Comment:
"collected less than ${FETAL_STORAGE_WEEKS} weeks ago." <-- add word "ago" to clarify this sentence.

Reply:
Done 👍